### PR TITLE
Updated Auth Connectors Examples

### PR DESCRIPTION
**Pull request by benarent**
As part of the upcoming Teleport Redesign, I looked back at our standard starter content for auth connectors. This PR makes `examples/resources/*` the canonical source of this content.  I've leveraged mkdocs.include module to reduce errors and this has been reflected in a couple of docs files. We'll plan on expanding it to more.

### CODE REVIEW

**Comment by stevenGravy** [comment]
Currently documentation examples has the SSO Connectors with examples of "Login with Example" which leads to "Login with Login with Example" since the "Login with" is a automatic prefix.   These appear to be fixed so the guidance is to put the name of the SSO provide only ("google", "okta",...) so it renders as "Login with Okta".

**Review by stevenGravy — COMMENTED** [review]
The examples/resources/oidc-connector.yaml should be changed to have just display: "Example" instead of "Login with Example".  Otherwise you get "Login with  Login with Example" rendered.

File: examples/resources/adfs-connector.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   # display allows to set the caption of the "login" button
   # in the Web interface
-  display: "Login with SSO"
+  display: "Login with Active Directory"
```
Comment (webvictim) [inline examples/resources/adfs-connector.yaml]: These automatically have "Login with" added to the beginning when shown in the web interface, so we shouldn't put it here or you end up with "Login with Login with Active Directory"

File: examples/resources/adfs-connector.yaml [hunk 2]
```diff
@@ -50,8 +50,8 @@ spec:
 
   attributes_to_roles:
     - name: "http://schemas.xmlsoap.org/claims/Group"
-      value: "teleadmins"
-      roles: ["admins"]
+      value: "admins"
```
Comment (webvictim) [inline examples/resources/adfs-connector.yaml]: Might want to make this "Administrators" so it's more obvious that it's referring to a Windows group name

File: examples/resources/adfs-connector.yaml [hunk 3]
```diff
@@ -50,8 +50,8 @@ spec:
...
+      roles: ["admin"]
     - name: "http://schemas.xmlsoap.org/claims/Group"
-      value: "teleusers"
-      roles: ["users"]
+      value: "users"
```
Comment (webvictim) [inline examples/resources/adfs-connector.yaml]: `Users` maybe for the same reason as above

File: examples/resources/admin.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      - list
+      - create
+      - read
+      - update
+      - delete
+    - resources:
+      - event
+      verbs:
+      - list
+      - read
```
Comment (webvictim) [inline examples/resources/admin.yaml]: Can we consider taking this giant list out? People tend to copy/paste it in and their admin roles get way more complicated to debug as a result.

File: examples/resources/oidc-connector.yaml [hunk 5]
```diff
@@ -1,3 +1,7 @@
+#
+# Example resource for an OIDC connector
+# We recommend using oidc for G Suite, Auth 0 and Keyclock
```
Comment (webvictim) [inline examples/resources/oidc-connector.yaml]: `We recommend using OIDC for G Suite, Auth0 and Keycloak`

File: examples/resources/oidc-connector.yaml [hunk 6]
```diff
@@ -7,19 +11,21 @@ spec:
   provider: example
 
   # This value will be used in the Web UI for "[login with example]" button
+  # button on the cluster login screen so it will say "Login with Google". 
+  # Teleport will provide custom CSS for 'Google' and 'Auth0'. 
   display: "Login with Example"
```
Comment (webvictim) [inline examples/resources/oidc-connector.yaml]: Again, this shouldn't be "Login with Example", just "Example" as we automatically add "Login with" in the web interface.

File: examples/resources/saml-connector.yaml [hunk 7]
```diff
@@ -8,17 +8,18 @@ metadata:
...
 spec:
-  # display allows to set the caption of the "login" button
-  # in the Web interface
-  display: "Login with SSO"
-  # acs is the Assertion Consumer Service URL. This should be the address of
-  # the Teleport proxy that your identity provider will communicate with.
-  acs: https://teleport-proxy.example.com:3080/v1/webapi/saml/acs
+  # connector display name that will be appended to the title of "Login with"
+  # button on the cluster login screen so it will say "Login with Okta"
+  display: Okta
```
Comment (webvictim) [inline examples/resources/saml-connector.yaml]: Weirdly we already fixed this one but none of the others...

**Review by webvictim — APPROVED** [review]

File: examples/resources/admin.yaml (hunk 4 above)
Comment (benarent) [reply examples/resources/admin.yaml]: Good idea. I'm going to update it, and add everything into the role reference.

File: examples/resources/saml-connector.yaml (hunk 7 above)
Comment (benarent) [reply examples/resources/saml-connector.yaml]: I've updated the rest now.

File: examples/resources/adfs-connector.yaml (hunk 2 above)
Comment (benarent) [reply examples/resources/adfs-connector.yaml]: ty. I didn't know that. I've updated it.

**Pull request merged**